cmd/osmosisd/cmd/root.go

```diff
@@ -389,7 +405,7 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 					}
 				})
 			}
-			return server.InterceptConfigsPreRunHandler(cmd, customAppTemplate, customAppConfig, tmcfg.DefaultConfig())
+			return server.InterceptConfigsPreRunHandler(cmd, customAppTemplate, customAppConfig, cometConfig)
```

Comment:
Need to open this up in an editor -- was this just a bug before that it was default config, not comet config?